lib/CConfigHandler.cpp

```diff
@@ -17,6 +17,7 @@
 VCMI_LIB_NAMESPACE_BEGIN
 
 SettingsStorage settings;
+SettingsStorage persistent;

```

Comment:
Perhaps more complete name? Especially since this is global variable.
```suggestion
SettingsStorage persistentStorage;
```

```suggestion
SettingsStorage persistentSettings;
```